news.xml

```diff
@@ -1,6 +1,119 @@
+      NixOS 20.09 released
+    </title>
+    <description>
+      <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/20.09-nightingale/nightingale.png">
+        <img class="inline" src="logo/nixos-logo-20.09-nightingale-lores.png" alt="20.09 Nightingale logo" with="100" />
+      </a>
+      <p>
+        Hey everyone, I'm <a href="https://github.com/jonringer">Jonathan Ringer</a>,
+        one of the release managers for 20.09. As promised, the latest stable
+        release is here: NixOS 20.09 “Nightingale” ✨.
+      </p>
+
+      <ul>
+        <li>
+          <a href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09">Release manual</a>
+          <ul>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-highlights">Highlights</a>
+            </li>
+            <li><a href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-new-services">New
+                Services</a></li>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-incompatibilities">Backward
+                Incompatibilities</a></li>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-notable-changes">Other
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="https://nixos.org/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="https://nixos.org/download.html#download-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        The 20.09 release was possible due to the efforts of <strong>1313</strong>
+        contributors in <strong>31282</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/fridh">Frederik Rietdijk</a>,
+        <a href="https://github.com/mic92">Jörg Thalheim</a>,
+        <a href="https://github.com/ma27">Maximilian Bosch</a>,
+        <a href="https://github.com/jonringer">Jonathan Ringer</a>,
+        <a href="https://github.com/jtojnar">Jan Tojnar</a>,
+        <a href="https://github.com/danieldk">Daniël de Kok</a>,
+        <a href="https://github.com/worldofpeace">WORLDofPEACE</a>,
+        <a href="https://github.com/flokli">Florian Klink</a>,
+        <a href="https://github.com/romildo">José Romildo Malaquias</a>.
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
+      </p>
+
+      <p>
+        This didn't stop us. In the last six months:
+      </p>
+
+      <ul>
+        <li>7349 packages were added,</li>
```

Comment:
i think the `,` don't add any useful information and don't belong in such list. it's not a sentence

Reply:
And remove the final period in the list.